### ISSUE TITLE
Get object error when created at tag/1.2.2

### ISSUE THREAD

**Issue by shino** [basho/riak_cs#410]
At branch gh300-multipart-upload-rebase1

```
(py27_base)shino@shino-xub-vbox$ git show | head -5                                                          riak_cs-ss-range-support/. [git:ss-range-support-trial1] -
commit 2454371e3ab1152e080875c711426366826747ff
Author: Scott Lystig Fritchie <slfritchie@snookles.com>
Date:   Wed Feb 6 11:08:57 2013 +0900

    Add Bucket & Key args to use unhashed names for 2I calculation
```

Steps to reproduce.
1. Start riak_cs at tag/1.2.2 and s3cmd put
2. Stop riak_cs
3. Start riak_cs at gh300-multipart-upload-rebase1
4. s3cmd get (get the object which was created at step 1.)
5. Error occurs

Error in console:

```
(riak-cs@127.0.0.1)1> 22:44:58.216 [error] gen_fsm <0.4410.0> in state prepare terminated with reason: no function clause matching proplists:get_value(multipart, undefined, false) line 222
22:44:58.219 [error] CRASH REPORT Process <0.4410.0> with 0 neighbours exited with reason: no function clause matching proplists:get_value(multipart, undefined, false) line 222 in gen_fsm:terminate/7 line 611
22:44:58.222 [error] gen_fsm <0.4411.0> in state waiting_update_command terminated with reason: no function clause matching proplists:get_value(multipart, undefined, false) line 222
22:44:58.228 [error] CRASH REPORT Process <0.4411.0> with 0 neighbours exited with reason: no function clause matching proplists:get_value(multipart, undefined, false) line 222 in gen_fsm:terminate/7 line 611
22:44:58.232 [error] Supervisor riak_cs_get_fsm_sup had child undefined started with {riak_cs_get_fsm,start_link,undefined} at <0.4410.0> exit with reason no function clause matching proplists:get_value(multipart, undefined, false) line 222 in context child_terminated
22:44:58.236 [error] webmachine error: path="/buckets/test-bucket/objects/test.txt"
{exit,{{function_clause,[{proplists,get_value,[multipart,undefined,false],[{file,"proplists.erl"},{line,222}]},{riak_cs_mp_utils,clean_multipart_unused_parts,2,[{file,"src/riak_cs_mp_utils.erl"},{line,74}]},{riak_cs_get_fsm,prepare,1,[{file,"src/riak_cs_get_fsm.erl"},{line,318}]},{riak_cs_get_fsm,prepare,2,[{file,"src/riak_cs_get_fsm.erl"},{line,144}]},{gen_fsm,handle_msg,7,[{file,"gen_fsm.erl"},{line,494}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,227}]}]},{gen_fsm,sync_send_event,[<0.4410.0>,get_manifest,infinity]}},[{gen_fsm,sync_send_event,3,[{file,"gen_fsm.erl"},{line,214}]},{riak_cs_wm_utils,ensure_doc,2,[{file,"src/riak_cs_wm_utils.erl"},{line,210}]},{riak_cs_wm_object,authorize,2,[{file,"src/riak_cs_wm_object.erl"},{line,47}]},{riak_cs_wm_common,authorize,2,[{file,"src/riak_cs_wm_common.erl"},{line,309}]},{riak_cs_wm_common,forbidden,2,[{file,"src/riak_cs_wm_common.erl"},{line,147}]},{webmachine_resource,resource_call,3,[{file,"src/webmachine_resource.erl"},{line,183}]},{webmachine_resource,do,3,[{file,"src/webmachine_resource.erl"},{line,141}]},{webmachine_decision_core,resource_call,1,[{file,"src/webmachine_decision_core.erl"},{line,48}]}]}
[{gen_fsm,sync_send_event,3,[{file,"gen_fsm.erl"},{line,214}]},{riak_cs_wm_utils,ensure_doc,2,[{file,"src/riak_cs_wm_utils.erl"},{line,210}]},{riak_cs_wm_object,authorize,2,[{file,"src/riak_cs_wm_object.erl"},{line,47}]},{riak_cs_wm_common,authorize,2,[{file,"src/riak_cs_wm_common.erl"},{line,309}]},{riak_cs_wm_common,forbidden,2,[{file,"src/riak_cs_wm_common.erl"},{line,147}]},{webmachine_resource,resource_call,3,[{file,"src/webmachine_resource.erl"},{line,183}]},{webmachine_resource,do,3,[{file,"src/webmachine_resource.erl"},{line,141}]},{webmachine_decision_core,resource_call,1,[{file,"src/webmachine_decision_core.erl"},{line,48}]}]
```

**Comment by shino** [comment]
memo: at tag 1.2.2, riak_cs_put_fsm, L. 311

```
prepare(State=#state{bucket=Bucket,
                     key=Key,
                     block_size=BlockSize,
                     content_length=ContentLength,
                     content_type=ContentType,
                     metadata=Metadata,
                     acl=Acl,
                     riakc_pid=RiakPid})
  [snip]
    Manifest =
        riak_cs_lfs_utils:new_manifest(Bucket,
                                         Key,
                                         UUID,
                                         ContentLength,
                                         ContentType,
                                         %% we don't know the md5 yet
                                         undefined,
                                         Metadata,
                                         BlockSize,
                                         Acl,
                                         undefined,   % prop
                                         ClusterID),
```

#lfs_manifest_v3.prop is not `proplist()` but `proplists() | undefined`.

**Comment by kuenishi** [comment]
According to today's discussion, Scott told me and Shino that the fix in #416 is good way to fix this. So this will be closed when fix in #416 is merged. Sort of duplicate, though.

**Comment by shino** [comment]
I confirmed successful GET at current master 28dac0025f166a18c86fc.
close.
